res/Sectors/M1105/Centrax.xml

```diff
@@ -38,7 +38,14 @@
     <Allegiance Code="HvFd">Hive Federation</Allegiance>
```

Comment:
Please don't define 4 character codes; new ones should go through the T5SS review process so we can agree on globally unique identifiers. Ones that are already defined shouldn't be repeated here. (I should make this an error.)

Reply:
Done, I missed that in a cleanup.